src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorLanguageServer.cs

```diff
@@ -69,11 +69,6 @@ protected override ILspServices ConstructLspServices()
             .AddOptions()
             .AddLogging();
 
-        if (_configureServer is not null)
-        {
-            _configureServer(services);
-        }
-
```

Comment:
I'm a bit worried this will change something that isn't well covered by tests, and we lose or change logging some how.

I think it would be better to not change product code in this PR, and just not use DI in the benchmark: Just create an instance of your `ClientNotifierService` and pass it in to the diagnostics endpoint you're creating.